### ISSUE TITLE
Update DCP 1.0 -> DCP 2.0 migration to reflect new schema.

### ISSUE THREAD

**Issue by maniarathi** [HumanCellAtlas/dcp-prototype#123]

**Comment by parthshahva** [comment]
PR is merged and dev db is now ready with the 2/18 schema. This work should now be unblocked. Let me know if you encounter any issues

**Comment by maniarathi** [comment]
In light of recent requirement changes, this ticket is no longer relevant and can be re-opened if circumstances change.

Context: https://docs.google.com/document/d/15HfenokYZGJBAd1CRRjGlZGxRtkV7PRnToe7Lsy_iBk/edit
